src/software/new_geom/util/distance.h

```diff
@@ -27,15 +27,6 @@ double distance(const Point &first, const Line &second);
  */
 double distance(const Point &first, const Point &second);
 
-/**
- * Finds the shortest distance between two Segments
- *
- * @param first
- * @param second
- * @return the shortest distance between first and second
- */
-double distance(const Segment &first, const Segment &second);
```

Comment:
Just checking that we're removing this because it's not used, and it means we can get rid of the boost stuff?

Reply:
yes that's right

Reply:
Pop quiz 👅 : Why is CI failing?

Reply:
^this is a fun riddle depending what you consider fun. I will give the answer at tmr's meeting

Reply:
OH this is _interesting_. I'm very curious to hear what this was.

Reply:
I'm going to guess network issues without looking at the log. I also am enticed by this riddle.